pkg/sink/initialization.go

```diff
@@ -51,6 +54,8 @@ type Args struct {
 	ElNamespace string
 	// Port is the port the Sink should listen on.
 	Port string
+	// ELType determines whether el functions as namespaced or clustered
```

Comment:
minor: would be clearer to make this a boolean flag e.g. IsClusterWide or IsMultiNameSpace